packages/cli/src/rpc/explain/buildContext.ts

```diff
@@ -44,34 +46,53 @@ export default async function buildContext(searchResults: SearchRpc.SearchResult
       formatOptions: { disableMarkup: true },
     });
     assert(typeof plantUML === 'string');
-    sequenceDiagrams.push(plantUML);
+    sequenceDiagrams.push({
+      type: ContextV2.ContextItemType.SequenceDiagram,
+      content: plantUML,
+      score: result.score,
```

Comment:
Possible to include a location here? This would be useful to link to an AppMap from the frontend.

Reply:
Yes good idea.